en/SUMMARY.md

```diff
@@ -68,11 +67,13 @@
 * [Robotics](robotics/README.md)
   * [Offboard Control from Linux](ros/offboard_control.md)
   * [ROS](ros/README.md)
-    * [ROS Installation on RPi](ros/raspberrypi_installation.md)
     * [MAVROS \(MAVLink on ROS\)](ros/mavros_installation.md)
     * [MAVROS offboard example](ros/mavros_offboard.md)
+    * [ROS with Gazebo Simulation](simulation/ros_interface.md)
+      * [Gazebo Octomap](simulation/gazebo_octomap.md)
```

Comment:
@TSC21 Any thoughts on "right" location for *Gazebo Octomap*? In theory it is a generic library that you could use with PX4 for environment mapping/obstacle avoidance - and if it were well integrated for that purpose it would be part of the main px4 concepts. 

In practice it is just an interesting tool for experimentation, and the documentation is tied very closely to both Gazebo and ROS.  Any thoughts on whether it would be better below Gazebo, where it is below ROS, or some other location?

Reply:
I would explicitly put it under ROS, as it is really tide more with ROS than with Gazebo (you can use outside sim).

Reply:
Thanks. I've moved it to same level and renamed to "OctoMap 3D Models with ROS" in sidebar and   "OctoMap 3D Models with ROS/Gazebo" in the page itself. YOu can see it here:  https://github.com/PX4/Devguide/pull/393/commits/8b5c8a74218a17a7061da6ceb77fff49a5753294